src/doc-editors/BackendServicesEditor.vue

```diff
@@ -119,7 +119,7 @@
     </div>
     <hr/>
     <div class="content"
-         v-if="selectedBackendService">
+         v-if="loadingDocCounter==0 && selectedBranch && selectedBackendService">
```

Comment:
```suggestion
         v-if="!loadingDocCounter && selectedBranch && selectedBackendService">
```

0 is falsy, so we can simplify the check